Sources/TensorFlow/Layers/Pooling.swift

```diff
@@ -43,10 +43,9 @@ public struct MaxPool1D<Scalar: TensorFlowFloatingPoint>: Layer {
     /// - Parameter input: The input to the layer.
     /// - Returns: The output.
     @differentiable
-    public func call(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
+    public func callAsFunction(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
         return maxPool2D(input.expandingShape(at: 1), filterSize: (1, 1, poolSize, 1),
-                         strides: (1, 1, stride, 1), padding: padding
-        ).squeezingShape(at: 1)
```

Comment:
This change is missing `.squeezingShape(at: 1)`

Reply:
Will look out for any errors like this in this PR. Wanted to push my changes here so it's easy to visually diff against master on Github.